packages/storybook/src/stories/grid-columns.stories.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { GridColDef, XGrid, GridColTypeDef } from '@material-ui/x-grid';
+import { GridColDef, XGrid, GridColTypeDef, ValueGetterParams } from '@material-ui/x-grid';
```

Comment:
It looks like a module we forgot to prefix with Grid. I would have expected:

```suggestion
import { GridColDef, XGrid, GridColTypeDef, GridValueGetterParams } from '@material-ui/x-grid';
```

cc @DanailH